_sections/videocalling-vscode.md

```diff
@@ -64,15 +64,19 @@ There are four main controls available whilst on the call. You can choose to mut
 
 #### CodeShare
 
-Selecting the share code option starts a collaboration session in your IDE. This means that any file you open will also open in the IDE of the other participant on the call, allowing you to edit together as you would in a google doc. You can even codeshare on files that aren't under source control (such as files ignored in Git) or files from outside the directory (such as external libraries’ source files). 
+Selecting the share code option opens up a modal, which shows the different projects you have open and the different methods of pair programming that are available. 
+
+![Share Code](/uploads/codeshare-gitlive-options.jpg "Share Code"){:class="screenshot"}
+
+Selecting the GitLive option is the quickest way to start a collaboration session in your IDE. This means that any file you open will also open in the IDE of the other participant on the call, allowing you to edit together as you would in a google doc. You can even codeshare on files that aren't under source control (such as files ignored in Git) or files from outside the directory (such as external libraries’ source files). 
 
 ![Share Code](/uploads/vscode-video-call-share-code.png "Share Code"){:class="screenshot"}
 
 By default, you will automatically follow each other's cursors when scrolling or switching files. You can disable following from within the IDE to work independently.
 
 ![Follow Cursor](/uploads/vscode-call-follow-cursor.jpeg "Follow Cursor"){:class="screenshot"}
 
-
+If you have Live Share installed, you can choose to use that to collaborate, instead of GitLive's proprietary code sharing solution. You can find more information about using Live Share [here](/docs/supportforlivesharecodewithme/){:class="internal-link"}. 
```

Comment:
Is that true? doesnt the live share option show based on the fact that the host has a vs code project open weather he has vs life share installed or not doesnt matter or am i wrong?

Reply:
Yes, check with Andrew. Need to have Live Share installed

Reply:
ok